plugins/modules/win_acl_inheritance.ps1

```diff
@@ -95,7 +95,9 @@ $regeditHives = @{
 }
 
 $pathQualifier = Split-Path -Path $path -Qualifier -ErrorAction SilentlyContinue
-$pathQualifier = $pathQualifier.Replace(':', '')
+if (':' -in $pathQualifier) {
```

Comment:
In PowerShell, `-in` (and `-contains`) are only used with arrays,  not strings.
```suggestion
if ($pathQualifier.Contains(':')) {
```

It's unclear to me how this change actually changes the behavior though. `$pathQualifier.Replace(':', '')` will already only do a replacement if the string contains a `:` and otherwise won't change it, so the conditional seems unnecessary.

Reply:
The existing code in main won't run against network paths due to checking for -Qualifier (which only works on local paths).

The condition was added solely to skip operations on $pathQualifier if the code in line 98 fails to set $pathQualifier (which is always with a network path)